### ISSUE TITLE
BOT Idea

### ISSUE THREAD

**Issue by Mothanick** [finndev/PokeBuddy#640]
we should have a grid to select where your bot has to stay inside so it doesn't get stuck somewhere far away if there is already a way to do something like that please let me know

**Comment by SynSyn** [comment]
yes please add a boundary system. Im tired of waking up in the morning to find my bot is super far away from the populated area.
